src/main/java/com/unascribed/fabrication/mixin/i_woina/end_portal_parallax/MixinWindow.java

```diff
@@ -0,0 +1,29 @@
+package com.unascribed.fabrication.mixin.i_woina.end_portal_parallax;
+
+import com.unascribed.fabrication.support.EligibleIf;
+import com.unascribed.fabrication.support.Env;
+import com.unascribed.fabrication.support.SpecialEligibility;
+import com.unascribed.fabrication.support.injection.FabInject;
+import net.minecraft.client.WindowEventHandler;
+import net.minecraft.client.WindowSettings;
+import net.minecraft.client.util.MonitorTracker;
+import net.minecraft.client.util.Window;
+import org.lwjgl.glfw.GLFW;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(Window.class)
+@EligibleIf(configAvailable="*.end_portal_parallax", envMatches=Env.CLIENT, specialConditions=SpecialEligibility.NOT_MACOS)
+public class MixinWindow {
+
+	@FabInject(at=@At(value="INVOKE", target="Lorg/lwjgl/glfw/GLFW;glfwCreateWindow(IILjava/lang/CharSequence;JJ)J"),
+			method="<init>")
+	private void Gl11support(WindowEventHandler eventHandler, MonitorTracker monitorTracker, WindowSettings settings, String videoMode, String title, CallbackInfo ci) {
+		GLFW.glfwWindowHint(GLFW.GLFW_OPENGL_PROFILE, GLFW.GLFW_OPENGL_ANY_PROFILE);
```

Comment:
Needs a FabConf check, or ideally to dep on Lib39 Deferral and report `FabConf.isEnabled("*.end_portal_parallax")` in the GLCompatVoter.

This fatally breaks some Linux setups in a way I've not yet been able to debug, but a fix for that is intended to go into Lib39 Deferral and I don't want to have to write it twice.

Reply:
i tried using Lib39 Deferral but it depended on 1.19 at which point i just copied it over.